php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -1002,30 +996,32 @@ class NDB_BVL_Feedback
      *
      * @author Evan McIlroy <evanmcilroy@gmail.com>
      *
-     * @return void
+     * @return int The number of updated threads
      */
     function openThread($feedbackID)
     {
+        if (empty($feedbackID)) {
```

Comment:
`int`s can't be empty. Might be nice to use type hints here instead.

Reply:
0 is empty for an int.

Reply:
Oops

Reply:
I still think type hints would be nice.

Alternatively
```suggestion
        if (! \Utility::valueIsPositiveInteger($feedbackID))) {
```

or

```suggestion
        if ($feedbackID < 1) {
```

I think these are nicer because it makes it clear we're dealing with a number. I won't block the PR over it though